content/pricing/index.textile

```diff
@@ -0,0 +1,119 @@
+
+- Messages <p>_(per million)_</p> := <p>The unit of data transfer throughout Ably.</p> <p>For example, if a user sends (publishes) a message and 10 users receive it (subscribed) that counts as 11 messages. Other operations also count towards the total message count, such as retrieving messages from history, or sending them through an integration.</p>
+- Active channels[^[*]^] <p>_(per million minutes)_</p> := <p>"Channels":/channels are used to logically separate message traffic into different topics, such as @account-debits@ and @account-credits@.</p> <p>For example, if you have three chat rooms open with users communicating in all of them for 2 hours then that would be 6 hours of channel time. The number of users attached to each channel does not influence the channel minutes.</p>
+- Connections <p>_(per million minutes)_</p> := <p>A "connection":/connect is established between your clients and Ably for realtime communication.</p> <p>For example, if you have 5 users that are all connected to Ably for exactly 10 minutes each, this counts as 50 minutes worth of connection time. If you have 50,000 users that are connected to Ably for 3 hours each, then this counts as 9,000,000 minutes.</p>
+
+<p><sup>[*]</sup> Channels are created on demand when clients attach to them. They remain active until there are no more realtime clients attached, approximately one minute has passed since the last client detached, and approximately one minute has passed since the last message was published to the channel.</p>
+
+The following table explains the limits related to the three primary units of consumption:
+
+|_. Limit |_. Free |_. Standard |_. Pro |_. Enterprise |
+| *Message rate (per second)*<p>_the maximum rate at which messages can be published and received across your account each second_</p> | 500 | 2,500 | 10,000 | Unlimited |
+| *Messages (per month)*<p>_the maximum number of messages that can be sent and received across your account each month_</p> | 6,000,000 | Unlimited | Unlimited | Unlimited |
+| *Concurrent channels*<p>_the maximum number of channels that are active simultaneously at any point_</p> | 200 | 10,000 | 50,000 | Unlimited |
+| *Concurrent connections*<p>_the maximum number of realtime clients connected to Ably simultaneously at any point_</p> | 200 | 10,000 | 50,000 | Unlimited |
+
+h3(#bandwidth). Bandwidth
+
+There is also a charge on additional bandwidth consumed.
+
+This is worked out based on the average message size of 5KiB multiplied by the number of messages consumed in the month. If your total bandwidth for the month is less than this value then there is no charge for bandwidth. Anything over this value is charged at $0.25 per GiB.
+
+h2(#packages). Packages
+
+Ably offers four different package types so that you can choose the one that best suits your needs:
+
+* "Free":/pricing/free: for experimenting with Ably, or building a proof of concept. No credit card required.
```

Comment:
Each of these bullet points have broken links. The trailing : at the end is considered part of the link, so it'll 404. You'll need to add a space between the url and the :